plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml

```diff
@@ -9,7 +9,7 @@
              
     <library 
           id="appengine-api"
-          group="appengine"
+          group="appengine-api"
           name="App Engine API"
           tooltip="%appengine.api.tooltip"
           siteUri="https://cloud.google.com/appengine/docs/java/" >
```

Comment:
Below here is the `export` attribute. It makes sense to change it to `false`.
```xml
       <libraryFile javadocUri="https://cloud.google.com/appengine/docs/java/javadoc/"
           export="true">
```

Reply:
done